pywinauto/controls/common_controls.py

```diff
@@ -657,6 +658,46 @@ def deselect(self):
     # Non PEP-8 alias
     Deselect = deselect
 
+    #-----------------------------------------------------------
+    def inplace_control(self, friendly_class_name=""):
+        """Return the editor HwndWrapper of the item
+
+        Possible ``friendly_class_name`` values:
+
+        * ``""``  Return the first appeared in-place control
+        * ``"friendlyclassname"``  Returns editor with particular friendlyclassname
+        """
+        # If currently editing in this item or some other
+        self.listview_ctrl.type_keys("{ENTER}")

```

Comment:
I think this could add potential side effect if we have an editable list box on a dialog with OK and Cancel buttons. This `{ENTER}` can close such a dialog. Maybe another workaround like click to the top-left corner would be better.